### DISCUSSION TITLE
Regular _search for DataSearch component

### DISCUSSION THREAD

**Discussion by vadymhimself** [appbaseio/reactivesearch#1633] [Q&A]
I am unable to set up the `_msearch` for my instance because of a weird issue with ES opendistro-for-elasticsearch/security-kibana-plugin#734

Can anyone suggest how to work this around?

**Comment by siddharthlatest** [comment]
@vadymhimself I'm hoping that this should be resolved now, at least I can't see an open issue in that repo.

ReactiveSearch uses `_msearch` (when querying from FE) to optimize the number of HTTP requests it needs to make. Ideally, any ES instance should allow this endpoint.

One possible workaround is to use appbase.io as a reverse-proxy (set `enableAppbase` prop in ReactiveBase) service to generate queries.
